src/main/java/de/dennisguse/opentracks/data/ContentProviderUtils.java

```diff
@@ -548,6 +548,35 @@ private Cursor getMarkerCursor(String[] projection, String selection, String[] s
         return contentResolver.query(MarkerColumns.CONTENT_URI, projection, selection, selectionArgs, sortOrder);
     }
 
+    public List<Marker> searchMarkers(Track.Id trackId, String query) {
+        String selection = null;
+        String[] selectionArgs = null;
+        String sortOrder = null;
+
+        if(query == null) {
```

Comment:
Just one more thing: formatting. :)

`if(` -> `if (` 
At leasts that what my AndroidStudio is doing on reformat.
All else looks okay.

Reply:
Sorry, I always forget to run this command 😅